shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/OnboardingScreen.kt

```diff
@@ -5,14 +5,14 @@ import com.mbta.tid.mbta_app.repositories.IAccessibilityStatusRepository
 enum class OnboardingScreen {
     Location,
     StationAccessibility,
-    HideMaps,
+    MapDisplay,
```

Comment:
This will crash the app on launch for people who have already gone through this screen, because we don’t handle errors in `OnboardingScreen.valueOf` when the `OnboardingRepository` is loading the completed screens.

Reply:
Oh Good catch. I may just leave it HideMaps then.